src/tools/illink/src/ILLink.RoslynAnalyzer/DataFlow/LocalDataFlowVisitor.cs

```diff
@@ -504,7 +505,7 @@ public override TValue VisitObjectCreation (IObjectCreationOperation operation,
 
 		public override TValue VisitFlowAnonymousFunction (IFlowAnonymousFunctionOperation operation, LocalDataFlowState<TValue, TValueLattice> state)
 		{
-			Debug.Assert (operation.Symbol.ContainingSymbol is IMethodSymbol);
+			Debug.Assert (operation.Symbol.ContainingSymbol is IMethodSymbol or IFieldSymbol);
```

Comment:
Why not property as well? What does this translate to:
```csharp
class Helper
{
    public Action<int> DoIt => (x) => Console.WriteLine(x);
}
```

I guess it could be a getter with the body as well...